addons/product_expiry/tests/test_stock_lot.py

```diff
@@ -615,3 +615,48 @@ def test_no_expiration_date(self):
         picking_out.action_confirm()
         picking_out.action_assign()
         self.assertEqual(picking_out.move_ids.state, 'assigned', 'Wrong state of move line.')
+
+    def test_no_lot(self):
+        """
+        Try to confirm a stock move with a quant that doesn't have lot
+        """
+        # Set the removal strategy to 'least_packages'
+        fefo_strategy = self.env['product.removal'].search(
+            [('method', '=', 'fefo')])
+        self.apple_product.categ_id.removal_strategy_id = fefo_strategy.id
+
+        lot_form = Form(self.LotObj)
+        lot_form.name = 'LOT001'
+        lot_form.product_id = self.apple_product
+        lot_form.company_id = self.env.company
+        apple_lot = lot_form.save()
```

Comment:
```suggestion
        apple_lot = self.LotObj.create({
            'name': 'LOT001',
            'product_id': self.apple_product.id,
            'company_id': self.env.company.id,
        })
```
No need to go through a `Form` for something so simple. :slightly_smiling_face: